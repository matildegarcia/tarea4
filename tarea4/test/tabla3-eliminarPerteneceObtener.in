# creo una tabla de tamanio 5
crearTTablaJugadores 5

# creo un jugador mario y la inserto en la tabla
# mario
crearPartida

crearJugada
3 1 2 3 1
agregarEnPartida

crearJugada
4 1 2 3 4
agregarEnPartida

crearJugador 3 42 Uruguay mario
crearFecha 1/11/2022
insertarJugadorEnTabla



# creo un jugador maria y la inserto en la tabla
# maria
crearPartida

crearJugada
24 1 3 2 1
agregarEnPartida

crearJugada
31 1 4 2 2
agregarEnPartida

crearJugada
21 3 1 3 2
agregarEnPartida

crearJugada
4 3 2 1 1
agregarEnPartida

crearJugador 1 31 Chile maria
crearFecha 7/4/1983
insertarJugadorEnTabla

# creo un jugador ariam y la inserto en la tabla
# como ariam tiene las mismas letras que maria se produce una colision
# ariam
crearPartida

crearJugada
4 4 1 2 1
agregarEnPartida

crearJugada
53 2 3 1 2
agregarEnPartida

crearJugada
6 2 1 3 2
agregarEnPartida

crearJugada
1 2 3 1 2
agregarEnPartida

crearJugador 6 4 Peru ariam
crearFecha 7/4/1984
insertarJugadorEnTabla

# imprimo la tabla
imprimirTTablaJugadores

# verifico que las 3 personas pertenezcan a la tabla y las obtengo
# 
# verifico que mario este en la tabla
perteneceATTablaJugadores mario
obtenerJugadorDeTTablaJugadores mario

# verifico que ariam este en la tabla
perteneceATTablaJugadores ariam
obtenerJugadorDeTTablaJugadores ariam

# verifico que maria este en la tabla
perteneceATTablaJugadores maria
obtenerJugadorDeTTablaJugadores maria 

# verifico que juan no este en la tabla
perteneceATTablaJugadores juan 

#
#
# Ahora probamos el eliminar
#
#

# elimino a maria de la tabla 
eliminarJugadorDeTTablaJugadores maria

# verifico que maria no esta en la tabla
perteneceATTablaJugadores maria 

# verifico que ariam sigue en la tabla
perteneceATTablaJugadores ariam 
obtenerJugadorDeTTablaJugadores ariam

# elimino a ariam y a mario de la tabla
eliminarJugadorDeTTablaJugadores ariam
eliminarJugadorDeTTablaJugadores mario 

# verifico que ninguno sigue en la tabla
perteneceATTablaJugadores ariam 
perteneceATTablaJugadores mario 

# libero la tabla
liberarTTablaJugadores

Fin