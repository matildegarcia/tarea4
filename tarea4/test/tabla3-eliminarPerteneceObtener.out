1>#  creo una tabla de tamanio 5.
2>3>#  creo un jugador mario y la inserto en la tabla.
4>#  mario.
5>6>7>8>9>10>11>12>13>#  creo un jugador maria y la inserto en la tabla.
14>#  maria.
15>16>17>18>19>20>21>22>23>24>25>26>27>#  creo un jugador ariam y la inserto en la tabla.
28>#  como ariam tiene las mismas letras que maria se produce una colision.
29>#  ariam.
30>31>32>33>34>35>36>37>38>39>40>41>42>#  imprimo la tabla.
43>Jugador 3: mario, Uruguay, 42 años
Jugada: 3 
Movimientos: Derecha Izquierda Arriba Derecha 
Jugada: 4 
Movimientos: Derecha Izquierda Arriba Abajo 
1/11/2022
Jugador 6: ariam, Peru, 4 años
Jugada: 1 
Movimientos: Izquierda Arriba Derecha Izquierda 
Jugada: 4 
Movimientos: Abajo Derecha Izquierda Derecha 
Jugada: 6 
Movimientos: Izquierda Derecha Arriba Izquierda 
Jugada: 53 
Movimientos: Izquierda Arriba Derecha Izquierda 
7/4/1984
Jugador 1: maria, Chile, 31 años
Jugada: 4 
Movimientos: Arriba Izquierda Derecha Derecha 
Jugada: 21 
Movimientos: Arriba Derecha Arriba Izquierda 
Jugada: 24 
Movimientos: Derecha Arriba Izquierda Derecha 
Jugada: 31 
Movimientos: Derecha Abajo Izquierda Izquierda 
7/4/1983
44>#  verifico que las 3 personas pertenezcan a la tabla y las obtengo.
45>#  .
46>#  verifico que mario este en la tabla.
47>El jugador con nombre mario está en la tabla.
48>Jugador 3: mario, Uruguay, 42 años
Jugada: 3 
Movimientos: Derecha Izquierda Arriba Derecha 
Jugada: 4 
Movimientos: Derecha Izquierda Arriba Abajo 
49>#  verifico que ariam este en la tabla.
50>El jugador con nombre ariam está en la tabla.
51>Jugador 6: ariam, Peru, 4 años
Jugada: 1 
Movimientos: Izquierda Arriba Derecha Izquierda 
Jugada: 4 
Movimientos: Abajo Derecha Izquierda Derecha 
Jugada: 6 
Movimientos: Izquierda Derecha Arriba Izquierda 
Jugada: 53 
Movimientos: Izquierda Arriba Derecha Izquierda 
52>#  verifico que maria este en la tabla.
53>El jugador con nombre maria está en la tabla.
54>Jugador 1: maria, Chile, 31 años
Jugada: 4 
Movimientos: Arriba Izquierda Derecha Derecha 
Jugada: 21 
Movimientos: Arriba Derecha Arriba Izquierda 
Jugada: 24 
Movimientos: Derecha Arriba Izquierda Derecha 
Jugada: 31 
Movimientos: Derecha Abajo Izquierda Izquierda 
55>#  verifico que juan no este en la tabla.
56>El jugador con nombre juan NO está en la tabla.
57>#  
.
58># 
.
59>#  Ahora probamos el eliminar.
60># 
.
61># 
.
62>#  elimino a maria de la tabla .
63>64>#  verifico que maria no esta en la tabla.
65>El jugador con nombre maria NO está en la tabla.
66>#  verifico que ariam sigue en la tabla.
67>El jugador con nombre ariam está en la tabla.
68>Jugador 6: ariam, Peru, 4 años
Jugada: 1 
Movimientos: Izquierda Arriba Derecha Izquierda 
Jugada: 4 
Movimientos: Abajo Derecha Izquierda Derecha 
Jugada: 6 
Movimientos: Izquierda Derecha Arriba Izquierda 
Jugada: 53 
Movimientos: Izquierda Arriba Derecha Izquierda 
69>#  elimino a ariam y a mario de la tabla.
70>71>72>#  verifico que ninguno sigue en la tabla.
73>El jugador con nombre ariam NO está en la tabla.
74>El jugador con nombre mario NO está en la tabla.
75>#  libero la tabla.
76>77>Fin.
